doc/md/CLI/welcome.md

```diff
@@ -0,0 +1,40 @@
+---
+id: Welcome to Atlas
```

Comment:
atlas is the project, which already has an intro page,i think this page presents the command line getting started instructions?

Reply:
👍